### PR TITLE
(PC-26095) feat(Venue): add link in contact block web

### PR DESCRIPTION
**Pull request by marineshaw**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-26095

## Checklist

I have:

- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [x] Written **unit tests** native (and web when implementation is different) for my feature.
- [x] Added a **screenshot** for UI tickets or deleted the screenshot section if no UI change
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion][1]

## Screenshots

**delete** _if no UI change_

| Platform         | Mockup/Before | After |
| :--------------- | :-----------: | :---: |
| iOS              |image|image|
| Android          |image|image|
| Phone - Chrome   |image|image|
| Desktop - Chrome |image|image|

[1]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/venue/components/ContactBlockNew/ContactBlockNew.web.tsx [hunk 1]
```diff
@@ -3,35 +3,43 @@ import React from 'react'
...
 import { ExternalSiteFilled } from 'ui/svg/icons/ExternalSiteFilled'
 import { PhoneFilled } from 'ui/svg/icons/PhoneFilled'
 
 export const ContactBlock: React.FC<{ venue: VenueResponse }> = ({ venue }) => {
   const { email, phoneNumber, website } = venue?.contact || {}
+  const { showErrorSnackBar } = useSnackBarContext()
+
+  const onOpenUrlError = () => {
+    showErrorSnackBar({
+      message: 'Une erreur est survenue',
```
Comment (voisinhugo) [inline src/features/venue/components/ContactBlockNew/ContactBlockNew.web.tsx]: On est sûr du wording ?

File: src/features/venue/components/ContactBlockNew/ContactBlockNew.native.test.tsx [hunk 2]
```diff
@@ -3,40 +3,51 @@ import React from 'react'
...
-const phoneNumber = '+33102030405'
-const website = 'https://my@website.com'
 
-const openMail = jest.spyOn(ContactHelpers, 'openMail')
-const openPhoneNumber = jest.spyOn(ContactHelpers, 'openPhoneNumber')
-const openUrl = jest.spyOn(NavigationHelpers, 'openUrl')
+const mockShowErrorSnackBar = jest.fn()
+jest.mock('ui/components/snackBar/SnackBarContext', () => ({
+  useSnackBarContext: () => ({
+    showErrorSnackBar: jest.fn((props: SnackBarHelperSettings) => mockShowErrorSnackBar(props)),
```
Comment (aliraiki) [inline src/features/venue/components/ContactBlockNew/ContactBlockNew.native.test.tsx:16]:
```suggestion
    showErrorSnackBar: mockShowErrorSnackBar,
```

File: src/features/venue/components/ContactBlockNew/ContactBlockNew.native.test.tsx [hunk 3]
```diff
@@ -77,6 +88,17 @@ describe('<ContactBlock/>', () => {
     })
   })
 
+  it('should show snackbar when error', async () => {
+    render(<ContactBlock venue={venueResponseSnap} />)
+    openUrlSpy.mockRejectedValueOnce(new Error('error'))
```
Comment (aliraiki) [inline src/features/venue/components/ContactBlockNew/ContactBlockNew.native.test.tsx:93]: Tu peux mettre le mock avant le render je pense

File: src/features/venue/components/ContactBlockNew/ContactBlockNew.tsx [hunk 4]
```diff
@@ -1,91 +1,63 @@
...
 
-  const onPressWebsite = useCallback(() => {
-    if (!website) return
-    analytics.logVenueContact({ type: 'website', venueId: venue.id })
-    openUrl(website)
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [website])
+  const onOpenUrlError = () => {
+    showErrorSnackBar({
+      message: 'Une erreur est survenue',
```
Comment (aliraiki) [inline src/features/venue/components/ContactBlockNew/ContactBlockNew.tsx:20]:
```suggestion
      message: 'Une erreur est survenue.',
```

**Review by voisinhugo — APPROVED** [review]

**Review by aliraiki — APPROVED** [review]

**Pull request merged**